projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -74,39 +76,75 @@ function getSuggestionFromOpenAI(
 		} );
 }
 
+function createPrompt( title = '', content = '', categoryNames = '' ) {
+	// If title is not added, we will only complete.
+	if ( ! title ) {
+		return content;
+	}
+
+	// Title, content and categories
+	if ( content && categoryNames.length ) {
+		return sprintf(
+			/** translators: This will be a prompt to OpenAI to generate a post based on the post title, comma-seperated category names and the last 240 characters of content. */
+			__( "This is a post titled '%1$s', published in categories '%2$s':\n\n … %3$s", 'jetpack' ), // eslint-disable-line @wordpress/i18n-no-collapsible-whitespace
+			title,
+			categoryNames,
+			content
+		);
+	}
+
+	// No content, only title and categories
+	if ( categoryNames.length ) {
+		return sprintf(
+			/** translators: This will be a prompt to OpenAI to generate a post based on the post title, and comma-seperated category names. */
+			__( "This is a post titled '%1$s', published in categories '%2$s':\n", 'jetpack' ), // eslint-disable-line @wordpress/i18n-no-collapsible-whitespace
+			title,
+			categoryNames
+		);
+	}
+
+	// Title and content
+	if ( content ) {
+		return sprintf(
+			/** translators: This will be a prompt to OpenAI to generate a post based on the post title, and the last 240 characters of content. */
+			__( "This is a post titled '%1$s':\n\n…%2$s", 'jetpack' ), // eslint-disable-line @wordpress/i18n-no-collapsible-whitespace
+			title,
+			content
+		);
+	}
+
+	/** translators: This will be a prompt to OpenAI to generate a post based on the post title */
+	return sprintf( __( 'Write content of a post titled "%s"', 'jetpack' ), title );
```

Comment:
This seems to break the build:

```
ERROR in editor-beta.js: Translator comments have gone missing for "Write content of a post titled \"%s\""
```

This could help:

```suggestion
	return sprintf(
		/** translators: This will be a prompt to OpenAI to generate a post based on the post title */
		__( 'Write content of a post titled "%s"', 'jetpack' ),
		title
	);
```